frontend/src/components/profile/profile.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import ProfileMenu from "./components/profile-menu/profile-menu";
+import ProfileInfo from "./components/profile-info/profile-info";
+import { useSelector } from 'react-redux';
+import { RootState } from 'common/types';
+
+
+import styles from './styles.module.scss';
+
+const Profile: React.FC = () => {
+  const { user } = useSelector(({ auth }: RootState) => ({
+  user: auth.user
+  }));
+  const editUserInfo = () => {
```

Comment:
```suggestion
  const handleUserInfoEdit = () => {
```
use handle for local component cb